README.md

```diff
@@ -517,6 +517,24 @@ Upload Lighthouse results as [action artifacts](https://help.github.com/en/actio
 uploadArtifacts: true
 ```
 
+#### `artifactName` (default: lighthouse-results)
+
+Filename of the action artifact when using `uploadArtifacts`.
+
+```yml
+artifactName: my-custom-name
+```
+
+#### `artifactRetentionDays` (default: undefined)
```

Comment:
Looking at the rest of the docs I'm now thinking this `default: undefined` isn't necessary. Thoughts?